source/sass/formassembly-override.scss

```diff
@@ -152,3 +161,114 @@ $input-font-size: 1.25rem !important;
 #tfa_494-D {
   margin-top: 0;
 }
+
+// DonateHelpPage form specific overrides
+.donate-help-page-content {
+  .wForm {
+    label {
+      font-family: $font-family-sans-serif;
+      color: $black !important;
+      font-size: 1.125rem;
+      margin-bottom: 0;
+    }
+
+    input[type="text"],
+    input[type="text"].validate-email,
+    textarea,
+    select {
+      border-radius: 0 !important;
+      border: 1px solid black !important;
+    }
+
+    input[type="file"] {
+      padding: 0 !important;
+      height: auto !important;
+
+      &::file-selector-button {
+        transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out;
+        font-family: $font-family-sans-serif;
+        background: $light-blue;
+        width: auto;
+        padding: 0rem 1.5rem;
+        font-size: 1rem;
+        border: none;
+        height: 2.25rem;
+
+        &:hover {
+          background: $blue;
+          cursor: pointer;
+        }
+      }
+    }
+
+    textarea {
+      height: 10em !important;
```

Comment:
`<textarea>` by deafult can be resized by users. Setting a fixed height for `<textarea>` would be confusing to users because they still see the resize control but dragging it around doesn't change the sizing due to our CSS override.

To improve the behaviours, we have three options here:

1. Give `textarea` a fixed `height` & add `resize: none;` to hide the resizing UI control.
2. Change this line to use `min-height` instead.
3. Don't add any height/size CSS overrides to `textarea`.

Reply:
Thanks for the explanation @mmmavis!

The height of the text box was updated per request of the stakeholder, however, I agree that users should be able to resize it if needed.

For that reason, I went with option 2, which is updating the `height` to `min-height`.